src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/CommPkg.cs

```diff
@@ -47,6 +47,7 @@ protected CommPkg()
             Status = status;
             System = system;
             ProjectId = project.Id;
+            RfocAccepted = false;
```

Comment:
is this always false when commPkg is created?

Reply:
Should be, yes. Should not be allowed to add a commpkg to invitation that has rfoc accepted. This is checked when creating/editing invitation